### PR TITLE
Reference types has finished, template update

### PR DESCRIPTION
**Pull request by sudo-bmitch**
Signed-off-by: Brandon Mitchell <git@bmitch.net>

The reference types working group delivered their PRs. This documents that, and updates the template with some missing sections.

### CODE REVIEW

File: WG-TEMPLATE.md [hunk 1]
```diff
@@ -24,6 +24,22 @@ The WG operates as an OCI Working Group under the [Open Container Initiative (OC
...
+
+## Stakeholders
+
+OCI Projects, non-OCI projects, or organizations sponsoring the working group and participating in the implementation and use case validation of the work done by the group.
+
+* {List proposed stakeholders.}
+
+## Related Issues/PRs
+
+* {List any issues that would be resolved by this working group}
```
Comment (samuelkarp) [inline WG-TEMPLATE.md:41]: nit: this is the only one that doesn't end with a period

```suggestion
* {List any issues that would be resolved by this working group.}
```

**Review by samuelkarp — APPROVED** [review]

File: WG-TEMPLATE.md (hunk 1 above)
Comment (sudo-bmitch) [reply WG-TEMPLATE.md]: Got it, thanks!

**Review by mikebrow — APPROVED** [review]
LGTM

**Comment by sudo-bmitch** [comment]
@opencontainers/tob are we waiting on anything to merge this?

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**